packages/projection/test/operators/Mappers/withHandles.test.ts

```diff
@@ -288,4 +290,40 @@ describe('withHandles', () => {
       expect(handles[0].latestOwnerAddress).toBe(maryAddress);
     });
   });
+
+  describe('subhandles', () => {
+    const project = (tx: Cardano.OnChainTx) =>
+      firstValueFrom(
+        of({
+          block: {
+            body: [tx],
+            header: mockProviders.ledgerTip
+          }
+        } as ProjectionEvent).pipe(withUtxo(), withCIP67(), withHandles({ policyIds: [handlePolicyId] }, logger))
+      );
+
+    it('adds parentHandle data for virtual subhandles that resolves to the parent address', async () => {
+      const { handles } = await project({
+        body: { outputs: [virtualSubHandleOutput] },
+        inputSource: Cardano.InputSource.inputs
+      } as Cardano.OnChainTx);
+
+      expect(handles).toHaveLength(1);
+
+      expect(handles[0].parentHandle).toBe(
+        'addr_test1qpadxfxylvy8p8wejlmt9wnesr2squgr524r77n7hz6yh3h34r3hjynmsy2cxpc04a6dkqxcsr29qfl7v9cmrd5mm89qqh563f'
+      );
+    });
+
+    it('adds parentHandle data for NFT subhandles that resolves to the parent subhandle', async () => {
```

Comment:
As far I understand, parent handle is not resolved by the mapper :thinking: 
```suggestion
    it('adds parentHandle data for NFT subhandles', async () => {
```

Reply:
addressed: 101ffed886a656361186adf3e31b20f62abd7089